addons/web_editor/controllers/main.py

```diff
@@ -212,6 +212,8 @@ def get_image_info(self, src=''):
                 attachment = request.env['ir.http']._xmlid_to_obj(request.env, url_segment)
             elif number_match: # numeric id
                 attachment = request.env['ir.attachment'].browse(int(number_match.group(1)))
+            else:
+                attachment = None
```

Comment:
maybe put a default `attachment = None` above the first `if`, to make the successive tests safe regardless of future refactorings?